jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
```

Comment:
Sorry, I forgot those functions...
Those functions depend on baxter, but don't depend on ARC task.
So I think those should be in util.l and no class name is needed.
Perhaps we can move those functions to baxtereus, but that's too exaggerated.